src/harness/util.py

```diff
@@ -376,3 +376,15 @@ def addGrantIdsToRequests(grant_ids, requests):
     requests: (list) list of requests, containing dictionaries.
   """
   addIdsToRequests(grant_ids, requests, 'grantId')
+
+def getCertificateFingerprint(certificate):
+  """ Get SHA1 hash of the input certificate.
+  Args:
+    certificate: certificate file
```

Comment:
Certificate   represents the fullpath of the certificate

Reply:
There was a similar discussion on the FAD.1 PR. I think using the relative path (or just the filename) is best because otherwise we all have to edit the config before we can run the test case with the WinnForum configuration.